R/landing_popup_module.R

```diff
@@ -0,0 +1,118 @@
+#' Landing Popup Module
+#'
+#' @description This module creates a landing welcome popup for `teal` applications.
```

Comment:
```suggestion
#' @description Creates a landing welcome popup for `teal` applications.
```

`@description` is not necessary